Backpack/src/main/res/drawable/bpk_switch_decoration.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <corners android:radius="@dimen/bpkBorderRadiusXl" />
```

Comment:
The largest border-radius we have so it always stays 50% regardless of switch height.